sdkcompat/v222/com/google/idea/sdkcompat/kotlin/KotlinLibraryConfiguratorForOldProjectModel.java

```diff
@@ -0,0 +1,24 @@
+package com.google.idea.sdkcompat.kotlin;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.roots.ModifiableRootModel;
+import org.jetbrains.kotlin.idea.configuration.KotlinJavaModuleConfigurator;
+
+/**
+ * #api203 The configurator specific for adding kotlin Library to old project model (2020.3 and
+ * 2021.1). This is a temporary workaround for b/198439707. TODO(b/198439707): Delete when old
+ * project model is not in use anymore.
+ */
+public class KotlinLibraryConfiguratorForOldProjectModel extends KotlinJavaModuleConfigurator {
```

Comment:
Please remove this class as it was removed in commit https://github.com/bazelbuild/intellij/commit/e7073ee518bf07ab402463ec0e533c9bf36960d3.